truss/contexts/local_loader/local_server_loader.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+import venv
+from pathlib import Path
+from typing import Any, List, Optional
+
+from truss.contexts.image_builder.serving_image_builder import ServingImageBuilder
+from truss.contexts.local_loader.docker_build_emulator import DockerBuildEmulator
+from truss.contexts.local_loader.truss_file_watcher import TrussFilesWatcher
+from truss.util.path import build_truss_target_directory
+
+
+class EnvBuilder(venv.EnvBuilder):
+    def __init__(self, venv_dir: Path, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.context: Any = None
+        self.venv_dir = venv_dir
+
+    def post_setup(self, context):
```

Comment:
What is this for? It will be good to have doc comments for the pubic functions here.

Reply:
I think this defines the post setup hook of the `venv.EnvBuilder` class to preserve the context – but I agree it's not clear without looking at the superclass.